Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2605,6 +2658,34 @@ namespace Babylon
                 auto& session{*XRSession::Unwrap(jsSession.Value())};
                 session.m_xr = std::move(nativeXr);
 
+                auto featureObject = info[1].As<Napi::Object>();
+                if (featureObject.Has("trackedImages"))
+                {
+                    const auto napiTrackedImages{featureObject.Get("trackedImages").As<Napi::Array>()};
+                    session.m_imageTrackingRequests.resize(napiTrackedImages.Length());
+
+                    // Create the tracked image buffer.
+                    for (uint32_t idx = 0; idx < napiTrackedImages.Length(); idx++)
+                    {
+                        const auto napiImageRequest{napiTrackedImages.Get(idx).As<Napi::Object>()};
+                        const auto napiImage{napiImageRequest.Get("image").As<Napi::Object>()};
+                        const auto napiBuffer{napiImage.Get("data").As<Napi::Uint8Array>()};
+                        const uint32_t bufferSize{(uint32_t) napiBuffer.ByteLength()};
+                        const uint32_t imageHeight{napiImage.Get("height").ToNumber().Uint32Value()};
+                        const uint32_t stride{bufferSize / imageHeight};
+                        const float estimatedWidth{napiImageRequest.Get("widthInMeters").ToNumber().FloatValue()};
+                        session.m_imageTrackingRequests[idx] =
+                        {
+                            (uint8_t *) napiBuffer.Data(),
```

Comment:
This cast should not be needed right? `Uint8Array::Data()` should already be of type `uint8_t*`.

Reply:
It looks like the deduced type of Uint8Array::Data is const char*  Here is the error I get in Android Studio without a cast:
Cannot initialize a variable of type 'uint8_t *' (aka 'unsigned char *') with an rvalue of type 'const unsigned char *'